pkg/system/init_windows.go

```diff
@@ -15,10 +15,10 @@ var (
 	containerdRuntimeSupported = false
 )
 
-// InitLCOW sets whether LCOW is supported or not
+// InitLCOW sets whether LCOW is supported or not. Requires RS5+
 func InitLCOW(experimental bool) {
 	v := GetOSVersion()
-	if experimental && v.Build >= 16299 {
+	if experimental && v.Build >= 17763 {
```

Comment:
there's cool constants for this now 😅 https://github.com/moby/moby/pull/39100

Reply:
I didn't realise that was merged yet. Updated.